manifests/09-prometheus-rules-imagestreams.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: imagestreams-rules
+  namespace: openshift-image-registry
+spec:
+  groups:
+  - name: imagestreams.rules
+    rules:
+    - expr: sum by (location, source) (image_registry_image_stream_tags_total)
+      record: imagestreamtags:count:sum
```

Comment:
does it make sense to prepend this with `imageregistry:`? i.e `imageregistry:imagestreamtags:count:sum`
additionally the other rule we're adding uses `_count` instead of `:count`. which one do we prefer?

Reply:
I prefer your approach. Have updated, PTAL.